Formula/nanomsgxx.rb

```diff
@@ -0,0 +1,44 @@
+class Nanomsgxx < Formula
+  desc "Nanomsg binding for C++11"
+  homepage "https://achille-roussel.github.io/nanomsgxx/doc/nanomsgxx.7.html"
+  url "https://github.com/achille-roussel/nanomsgxx/archive/0.2.tar.gz"
+  sha256 "116ad531b512d60ea75ef21f55fd9d31c00b172775548958e5e7d4edaeeedbaa"
+
+  option "with-debug", "Compile with debug symbols"
```

Comment:
Have you tried enabling this option and verified that the debug symbols are there as you expect?